apps/order/serializers.py

```diff
@@ -24,13 +24,29 @@ def validate_book(self, book):
 
 class CreateOrderFromCartSerializer(CreatedUpdatedBaseSerializer, serializers.ModelSerializer):
 
+    cart_item_ids = serializers.ListField(
+        child=serializers.IntegerField(required=True)
+    )
```

Comment:
Maybe use a custom field for better graphql schema generation.


**config/serializers.py**
```python3
class IntegerIDField(serializers.IntegerField):
    """
    This field is created to override the graphene conversion of the integerfield -> graphene.ID
    check out utils/graphene/mutation.py
    """
    pass


def IdListField(**args):
    return serializers.ListField(
        child=IntegerIDField(**args),
        default=list,
        allow_empty=True,
    )
```

**graphene/mutations.py**
```python3
..
@get_graphene_type_from_serializer_field.register(IntegerIDField)
..
def convert_serializer_field_to_id(field):
    return graphene.ID
```

Reply:
@thenav56  Added this

Reply:
Where? you are still using this.

```
serializers.ListField(
        child=serializers.IntegerField(required=True)
    )
```

Reply:
Oh no, Fixed